README.md

```diff
@@ -88,6 +101,31 @@ box. There you should put the URL to the application's lead resource
 
 ![webhook configuration](images/webhook.png)
 
+You can now purchase new numbers from Twilio, associate them with a lead source,
+and set up call forwarding from the dashboard.
+
+To add a new number press the "Search" button on the main dashboard. You can optionally
+select an area code to find a number in that particular area.
+
+![phone number search](images/phone-search.png)
+
+After you click "Search", you will be shown a list of available Twilio numbers that you
+can purchase. To select and purchase a number, click the "Purchase" button next
+to any of the listed available phone numbers.
+
+![available numbers view](images/purchase-number.png)
+
+You will then be redirected to a form where you can label the Lead Source and
+set up call forwarding. Now, when someone calls the number you have just purchased,
+it will be forwarded to the number you configure under "Forwarding number".
+
+![available numbers view](images/setup-lead.png)
+
+Now, when someone calls the number you purchased and labeled, the call will display
+in your dashboard as having been generated from that specific lead.
+
+![main dashboard view](images/main-dashboard.png)
+
```

Comment:
This visual guide is super helpful since I've never touched this part of console 🙌